src/GraphQL/Resolver/UserResolver.php

```diff
@@ -46,9 +48,13 @@ public function resolveUser($args)
         }
     }
 
-    public function resolveUserById($userId)
+    public function resolveUserById($userId): ?User
     {
-        return $this->userService->loadUser($userId);
+        try {
+            return $this->userService->loadUser($userId);
+        } catch (NotFoundException $e) {
+            return null;
```

Comment:
The handling is okay. But we may want to add a warning to the response payload: https://github.com/overblog/GraphQLBundle/blob/master/docs/error-handling/index.md.

Reply:
Done in https://github.com/ezsystems/ezplatform-graphql/pull/131/commits/7369cd09e96761c7de7efd9fd31fcd57c5bdb710